src/utils/ethereumUtils.js

```diff
@@ -8,15 +8,19 @@ import {
   toChecksumAddress,
 } from 'ethereumjs-util';
 import { hdkey } from 'ethereumjs-wallet';
-import { find, get, isEmpty, matchesProperty, replace, toLower } from 'lodash';
-import { NativeModules } from 'react-native';
+import {
+  find,
+  get,
+  isEmpty,
+  isString,
+  matchesProperty,
+  replace,
+  toLower,
+} from 'lodash';
+import { Linking, NativeModules } from 'react-native';
 import { ETHERSCAN_API_KEY } from 'react-native-dotenv';
 import URL from 'url-parse';
-import {
-  DEFAULT_HD_PATH,
-  identifyWalletType,
-  WalletLibraryType,
-} from '../model/wallet';
+import { getNetwork } from '@rainbow-me/handlers/localstorage/globalSettings';
```

Comment:
We can use network from redux store and not need to read from lcoalstorage every time for this! (And thereby get rid of await / async)

Reply:
okie how about now https://github.com/rainbow-me/rainbow/pull/1552/commits/e2219f58f9a1b6ab94fa24cd4c66d9c0414fc784